src/Pos/Context/Context.hs

```diff
@@ -149,6 +141,7 @@ data NodeContext ssc = NodeContext
     -- ^ Settings for global Txp.
     , ncConnectedPeers      :: !ConnectedPeers
     -- ^ Set of peers that we're connected to.
+    , ncNetworkConfig       :: !(NetworkConfig KademliaDHTInstance)
```

Comment:
The parameter to `NetworkConfig` is passed along to its `Topology` field, which ignores it in case of topologies which don't use kademlia/p2p. Check out `infra/Pos/Network/Types.hs`